<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>853</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Pretendard Variable Medium</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>계좌번호</string>
    </property>
   </widget>
   <widget class="QComboBox" name="accComboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>150</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>371</width>
      <height>312</height>
     </rect>
    </property>
    <property name="title">
     <string>주문서</string>
    </property>
    <widget class="QPushButton" name="cancelPushButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>260</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>취   소</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>70</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>수           량</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="sellPushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>매   도</string>
     </property>
    </widget>
    <widget class="QComboBox" name="gubunComboBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>111</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>종    목    명</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>70</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>가           격</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>70</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>원주문번호</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="buyPushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>매   수</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="searchItemTextEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>151</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="priceSpinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>151</width>
       <height>33</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
    <widget class="QTextEdit" name="itemCodeTextEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>151</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="ordernumberTextEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>151</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="searchItemButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>조      회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changePushButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>정   정</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>70</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>종 목 코 드</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="volumeSpinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>151</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>70</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>거 래 구 분</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="tradeGubunComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>151</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>371</width>
      <height>312</height>
     </rect>
    </property>
    <property name="title">
     <string>차트</string>
    </property>
    <widget class="QTextEdit" name="candlenumberTextEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>104</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>351</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="chartLayout"/>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>50</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>봉개수</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="chartPushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>671</width>
      <height>312</height>
     </rect>
    </property>
    <property name="title">
     <string>잔고확인</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>651</width>
       <height>191</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>계좌잔고</string>
      </attribute>
      <widget class="QTableWidget" name="stockListTableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>645</width>
         <height>163</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>미체결주문</string>
      </attribute>
      <widget class="QTableWidget" name="outstandingTableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>645</width>
         <height>163</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>651</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>총매입금액</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>추정예탁자산</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>총평가손익금액</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="balanceAssetLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>총평가금액</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="totalBuyingPriceLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="currentTotalPriceLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="totalEstimateProfitLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>671</width>
      <height>272</height>
     </rect>
    </property>
    <property name="title">
     <string>조건식/자동매매설정</string>
    </property>
    <widget class="QComboBox" name="autoTradeGubunComboBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="conditionSearchPushBox">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>40</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>조건검색시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="autoTradePushBox">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>220</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>자동매매시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addAutoTradePushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>추      가</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeAutoTradePushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>220</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>제      거</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>조건식 작동시간</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>매수/매도 구분</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="endTimeEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="conditionTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="startTimeEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>30</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>~</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="autoTradeConditionTableWidget">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>231</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>671</width>
      <height>162</height>
     </rect>
    </property>
    <property name="title">
     <string>조건검색목록</string>
    </property>
    <widget class="QTableWidget" name="conditionItemTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>651</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>330</y>
      <width>370</width>
      <height>160</height>
     </rect>
    </property>
    <property name="title">
     <string>조건</string>
    </property>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>10</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>원</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="realGubunComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>111</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>매 수 금 액</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="enableMoneyTextEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>151</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>70</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>매 수 구 분</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
